models/sipnet/R/write_restart.SIPNET.R

```diff
@@ -130,6 +132,8 @@ write_restart.SIPNET <- function(outdir, runid, start.time, stop.time, settings,
                                            settings = settings,
                                            run.id = runid,
                                            inputs = inputs,
-                                           IC = analysis.save.mat))
+                                           IC = analysis.save.mat,
+                                           obs_time=obs_time,
+					                                 update_phenology=update_phenology))
```

Comment:
Nit: whitespace alignment
```suggestion
                                           obs_time=obs_time,
					   update_phenology=update_phenology))
```